### ISSUE TITLE
20% of my screen is covered

### ISSUE THREAD

**Issue by Zer0mniac** [ScarletStudy/DGS1-Android-Release#39]
### About your device

**Device Manufacturer / Model or Emulator / Version??**
Samsung Galaxy S9
**Android Version?**
9.0.0
**Are you using a Custom ROM?** (If you don't know what this means, then most likely no)
NO
**Is your device rooted?** (If you don't know what this means, then most likely no)
No
### Your issue

**Please describe the issue you are experiencing:**
After applying the patch for "The Great Ace Attorney" 20% of my screen is now covered by the bar that shows the episode title and the clock. so my game doesn't use my whole screen
![Screenshot_20190417-030537_The Great Ace Attorney](https://user-images.githubusercontent.com/34463190/56281548-c0233c00-60c1-11e9-8526-0754e29c5304.jpg)

**Comment by Floflo81** [comment]
This probably can't be avoided, since afaik the game is designed for 16:9 screens.
I think the original unpatched game works exactly the same. Can you check?

**Comment by wabberz** [comment]
This is the intended behaviour of the game to preserve the aspect ratio of the 3DS screen.

**Comment by Zer0mniac** [comment]
@floflo81 yeah I checked the unpatched game and it utilizes the whole screen.

Lemme also say that before the patch for ep 5 it was also using the whole screen as well

**Comment by eckesru** [comment]
I can confirm confirm it is an issue from the Patch. The original japanese Version unpatched does NOT has this issue. Also before the case 5 update everything was fine for me. 

Samsung Galaxy S8

**Comment by jamfor999** [comment]
Can confirm I'm having the same issue on my Galaxy S8.

I'm not sure I understand why "preserving the 3DS aspect ratio" is suddenly important. This is the Android version we are playing, NOT the 3DS one. We expect 16:9 or greater support surely... The previous implementation worked fine and made the display really nice, and this is objectively awful. The issue of the massive bar is not occurring in the unmatched version, and I'm pretty certain it was NOT happening in the episode 4 release. If this is a new thing for episode 5, why has this been decided? It is horrible!

Obviously you guys do whatever you want, it's your translation patch and it's not like we are paying for it! Just seems a weird choice, if it really is intentional...

**Comment by jf352** [comment]
Is this being looked into @wabberz since this is obviously inconsistent with the previous patch and also the unmatched game?

**Comment by malucard** [comment]
It's important because it's how the game was made. It has nothing to do with the patch; every AA port for Android has a bar like that because the game wasn't designed with very wide screens in mind (they are just ports of 3DS games). 16:9 screens also have the bar (though smaller), because the 3DS ratio is a bit narrower. In fact AAI, AAI2 and AJ preserve the 4:3 ratio of the DS which makes the bars even larger.

edit: If it wasn't in the unpatched game, could you send a screenshot?

**Comment by Zer0mniac** [comment]
I'm just saying if it was ALWAYS like this I wouldn't have made this post. I can assure you the bar was never that long before the patch but uhh.. as of right now I don't currently own an android anymore so I can't really send any screenshots at the moment haha
